# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dash/tempfile-fix.patch
# Copyright (C) 2006 - 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- dash-0.5.7/src/mkbuiltins.orig	2011-03-15 08:18:06.000000000 +0100
+++ dash-0.5.7/src/mkbuiltins	2011-11-26 01:57:52.660104276 +0100
@@ -35,7 +35,7 @@
 #
 #	@(#)mkbuiltins	8.2 (Berkeley) 5/4/95
 
-tempfile=tempfile
+tempfile=mktemp
 if ! type tempfile > /dev/null 2>&1 && ! type mktemp > /dev/null 2>&1; then
 	_my_tempfile()
 	{
